_guides/packages/building.md

```diff
@@ -484,6 +484,24 @@ After importing the package, the new task is automatically assigned to the worke
 </p>
 </div>
 
+## Options
+### disable_events
+The **disable_events** option prevents bot behaviors or automations from triggering on new records. 
+
+<pre style="max-height:29.5em;">
+<code class="language-json">
+        "configure": {
+            "prompts": [],
+            "placeholders": [],
+            "options": {
+                "disable_events": true
+            }
+        }
+</code>
+</pre>
+This can be useful, for example, when importing historical records from another system where you don't want workflows for autoresponders or customer 
+surveys to trigger.
```

Comment:
@cohencor-m You don't need to manually wrap text in the Markdown source. Let phpStorm auto-wrap for you visually. Long lines are fine in the repo.